include/nbl/builtin/hlsl/emulated_float64_t.hlsl

```diff
@@ -625,7 +621,7 @@ namespace impl
         emulated_float64_t flipSign()
         {
             const uint64_t flippedSign = ((~data) & 0x8000000000000000ull);
-            return createEmulatedFloat64PreserveBitPattern(flippedSign | (data & 0x7FFFFFFFFFFFFFFFull));
+            return create(flippedSign | (data & 0x7FFFFFFFFFFFFFFFull));

```

Comment:
1. sign flip can be achieved with just a `^0x80...0ull`
2. you should not have a `create()` (a.k.a. constructor) that takes a literal bitpattern and works as a reinterpret_cast, add a `bit_cast` specialization for this and use that throughout the code instead